src/test/java/com/flightstats/hub/metrics/MetricsConfigTest.java

```diff
@@ -12,161 +15,26 @@
     public void testDefaultValues_defaultTags() {
         MetricsConfig metricsConfig = MetricsConfig
                 .builder()
+                .buildWithDefaults()
                 .build();
-        assertEquals(metricsConfig.getRole(), "");
-        assertEquals(metricsConfig.getTeam(), "");
-        assertEquals(metricsConfig.getEnv(), "dev");
-        assertEquals(metricsConfig.getAppVersion(), "local");
-        assertEquals(metricsConfig.getInfluxdbProtocol(), "http");
-        assertEquals(metricsConfig.getInfluxdbHost(), "localhost");
-        assertEquals(metricsConfig.getInfluxdbPort(), 8086);
-        assertEquals(metricsConfig.getInfluxdbPass(), "");
-        assertEquals(metricsConfig.getInfluxdbUser(), "");
-        assertEquals(metricsConfig.getInfluxdbDatabaseName(), "hubmain");
-        assertEquals(metricsConfig.getClusterTag(), "local-dev");
-        assertEquals(metricsConfig.getReportingIntervalSeconds(),  15);
+
+        String clusterTag = HubProperties.getProperty("cluster.location", "local") +
+                "-" +
+                HubProperties.getProperty("app.environment", "dev");
+
+        assertEquals(HubProperties.getProperty("metrics.tag.role", "hub"), metricsConfig.getRole());
```

Comment:
Tests like this always bug me just a little bit. It'll go away once we clean up the config design, so I'm cool with it for now.